### DISCUSSION TITLE
Filtering Card

### DISCUSSION THREAD

**Discussion by jman203** [vasqued2/ha-teamtracker-card#85] [General]
What ways have people found to show/hide this card? Currently, I use this card within a conditional card and have it show when the state of the team entity is not equal to NOT_FOUND. Overall this works well. However, I'd like for the card to not show if the game has been over for 2-3 days or isn't starting for 2-3 days. Currently I have cards showing with the games not starting for 3 months or have been over for 2-3 months.

**Comment by jman203** [comment]
I've figured out a good way to do this. You will need to create a template sensor to subtract the current date from the date attribute of the team sensor. This will give you the number of days, which can either be a negative number (if the game is upcoming) or a positive number (if the game has happened). You can then use this as part of the conditional card to show/hide the team card. Below is the template sensor I used.

This is a great integration and card you've put together. Thank you for your work.

```
"{{ (now().date() - ( state_attr('sensor.your_team_sensor', 'date' ) | as_datetime | as_local).date()).days }}"
```

**Comment by vasqued2** [comment]
Thanks for sharing!

**Comment by JeroenvanA** [comment]
Thanks!
